crates/bitcoin/src/types.rs

```diff
@@ -316,11 +317,16 @@ pub struct Transaction {
 #[cfg_attr(test, mockable)]
 impl Transaction {
     pub fn tx_id(&self) -> H256Le {
```

Comment:
You probably won't want to do this but I think it'd be good to feature gate this(and `hash` below) so we can be sure we don't use it in parachain production

Reply:
Agree but I did start to do this and it incurs a lot of changes, plus I removed some extrinsics in #1022 which use these methods so I would be inclined to wait until both PRs are merged then I can make the necessary changes to feature-gate those and the parser like you requested before.